src/no/runsafe/UserControl/database/PlayerDatabase.java

```diff
@@ -75,8 +75,11 @@ public ISchemaUpdate getSchemaUpdateQueries()
 			"ALTER TABLE player_db ADD COLUMN ban_by_uuid VARCHAR(36) NULL;",
 			// Convert banningPlayer usernames to UUIDs
 			"UPDATE IGNORE `player_db` SET `ban_by_uuid` = " +
-				"(SELECT `uuid` FROM `player_db` WHERE `name`=`player_db`.`ban_by`) " +
-				"WHERE `ban_by` IS NOT NULL;",
+				"(SELECT `uuid` FROM `player_username_log` WHERE `name`=`player_db`.`ban_by`) " +

```

Comment:
won't this table potentially have multiple rows for a given username? what if someone changed back to an old name, how was that handeled?

Reply:
If multiple people with the same UUID have had the same username this could give multiple results, I don't expect that to be an issue due to the limited number of users who have ever had /ban permissions but I can add a check just incase